Dockerfile

```diff
@@ -2,13 +2,43 @@ FROM node:14 AS development
 
 ENV USERNAME timetracker
 ENV HOME /home/${USERNAME}
+ENV CHROME_BIN /opt/google/chrome/google-chrome
+#Essential tools and xvfb
+RUN apt-get update && apt-get install -y \
+    software-properties-common \
+    unzip \
+    curl \
+    wget \
+    xvfb
+
+#Chrome browser to run the tests
+ARG CHROME_VERSION=65.0.3325.181
```

Comment:
Is this argument being used at some point?

Reply:
Yes, this environmental variable it's used to run Chrome Headless. This is an example in how it's used: https://qxf2.com/blog/building-your-own-docker-images-for-different-browser-versions/